modules/optimizing-mtu-networking.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/optimizing-networking.adoc
+
+[id="optimizing-mtu_{context}"]
+= Optimizing the MTU for your network
+
+There are two important maximum transmission units (MTUs): the network interface card (NIC) MTU and the SDN overlay's MTU.
+
+The NIC MTU is only configured at the time of {product-title} installation. The MTU Omust be less than or equal to the maximum supported value of the NIC of your network. If you are optimizing for throughput, choose the largest possible value. If you are optimizing for lowest latency, choose a lower value.
+
+The SDN overlay's MTU must be less than the NIC MTU by 50 bytes at a minimum. This accounts for the SDN overlay header. So, on a normal ethernet network, set this to `1450`. On a jumbo frame ethernet network, set this to `8950`.
+
+[NOTE]
+====
+This 50 byte overlay header is relevant to the OpenShift SDN. Other SDN solutions might require the value to be more or less.
+
+You must change the MTU size on all masters and nodes that are part of the {product-title} SDN. Also, the MTU size of the tun0 interface must be the same
```

Comment:
This is all handled internally now; Previously in OCP 3.x it was somewhat manual.

So I'd omit entirely the second paragraph. No one can actually do that anymore.